code/game/objects/items/devices/flashlight.dm

```diff
@@ -147,14 +150,76 @@
 
 /obj/item/device/flashlight/pen
 	name = "penlight"
-	desc = "A pen-sized light, used by medical staff."
+	desc = "A pen-sized light, used by medical staff to check the condition of eyes, brain and cosncience of patients."
```

Comment:
conscience is both misspelled and the wrong word to use here
also oxford comma

Reply:
Yes, will replace it with a more decent line